cmd/fabric-ca-server/config.go

```diff
@@ -576,6 +576,30 @@ func (s *ServerCmd) configInit() (err error) {
 		return err
 	}
 
+	// Read operations tls files
+	if s.myViper.GetString("operations.tls.enabled") == "true" {
+		cf := s.myViper.GetString("operations.tls.cert.file")
+		if util.FileExists(cf) {
+			if filepath.IsAbs(cf) {
+				s.cfg.Operations.TLS.CertFile = cf
+			} else {
+				s.cfg.Operations.TLS.CertFile = filepath.Join(filepath.Dir(s.cfgFileName), cf)
+			}
+		} else {
+			return errors.New("Failed to read certificate file")
```

Comment:
Error messages are lower case; messages should include context

https://github.com/golang/go/wiki/CodeReviewComments#error-strings